apps/meeting/src/utils/image.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export async function createColorBlob(): Promise<Blob> {
```

Comment:
Could we add a parameter for `hexColor: string`. So this API can create color blob based on input?

Reply:
Could refer to this? https://codepen.io/xuesichao/pen/GRbvpOZ
I think we can remove the gradient to make it simple.